convert2rhel/unit_tests/__init__.py

```diff
@@ -107,6 +109,21 @@ def safe_repr(obj, short=False):
     return result[:_MAX_LENGTH] + " [truncated]..."
 
 
+def get_pytest_mark(request, mark_name):
+    """
+    Get a pytest mark from a request.
+
+    The pytest API to retrieve a mark changed between RHEL6 and RHEL7.  This function is
+    a compatibility shim to retrieve the value.
```

Comment:
```suggestion
    a compatibility shim to retrieve the value.
    
    Use this function instead of pytest's `request.node.get_closest_marker(mark_name)` so that it will work on all versions of RHEL that we are targeting.
    .. seealso::
        * `pytest's get_closest_marker() function which this function wraps <https://docs.pytest.org/en/stable/reference/reference.html#pytest.nodes.Node.get_closest_marker>`
        * `A technique you might use where you would need to use this function to retrieve a mark's value. <https://docs.pytest.org/en/stable/how-to/fixtures.html#using-markers-to-pass-data-to-fixtures>`
```